tests/mbt/model/ccv_model.qnt

```diff
@@ -235,6 +239,23 @@ module ccv_model {
         stepCommon
     }
 
+    // Runs epochLength many blocks on the provider.
+    // The first block will start all consumers in consumersToStart and stop all consumers in consumersToStop,
+    // and advance time by timeAdvancement - ((epochLength-1) * Seconds).
+    // The rest of the blocks will not start or stop any consumers, and will advance time by 1 second each.
+    run EndProviderEpoch(
```

Comment:
This is only used for tests, right? Could you rename it accordingly or make it clear in the comment.